docs/README.md

```diff
@@ -1,4 +1,4 @@
-![NeoForged Logo](assets/neoforged_logo.png)
+![NeoForged Logo](../../NeoForge2/docs/assets/neoforged_logo.png)
```

Comment:
I don't think you meant to do this. 😄

Reply:
How did I not notice that.

Reply:
This isse is fixed in this commit (https://github.com/neoforged/NeoForge/pull/232/commits/66c9d8021cdfe8482abfabd77d33d1a2205442c1).